lib/base/utility.cpp

```diff
@@ -1310,6 +1310,13 @@ bool Utility::PathExists(const String& path)
 	return fs::exists(fs::path(path.Begin(), path.End()), ec) && !ec;
 }
 
+time_t Utility::GetFileCreationTime(const String& path)
+{
+	namespace fs = boost::filesystem;
+
+	return fs::last_write_time(static_cast<fs::path>(path));
```

Comment:
Why a static cast and not a function style cast?

Reply:
Because I want to cast it explicitly in a C++ way and not in `c-style`.

Reply:
Are **function style** casts **C style**?